src/components/modal/main/settings/m-settings.stories.ts

```diff
@@ -0,0 +1,20 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Does this component need a story? Since this component is only used for showcasing a modal